cmd/main.go

```diff
@@ -153,7 +153,7 @@ func getUserNamespaces(e *echo.Echo, nameReq labels.Requirement) ([]core.Namespa
 	}
 	req, _ := labels.NewRequirement("konflux.ci/type", selection.In, []string{"user"})
 	selector := labels.NewSelector().Add(*req)
-	selector.Add(nameReq)
+	selector = selector.Add(nameReq)
```

Comment:
What is the impact of this change? 
can you show  a log example of the difference in `selector` between those?

Reply:
Some extra context:
before
```
line155: Selector value: konflux.ci/type in (user)
line156: Selector value: konflux.ci/type in (user)
exclude-2)
```

after
```
line155: Selector value: konflux.ci/type in (user)
line156: Selector value: konflux.ci/type in (user),kubernetes.io/metadata.name in (in-test-1,in-test-2)
```

Reply:
It does seem to be the right way to use it, good catch!

Reply:
Okay. Added